### PR TITLE
(IMAGES-589) Add support for RHEL 7 AArch64

### PR DESCRIPTION
**Pull request by caseywilliams**

### CODE REVIEW

File: lib/beaker-hostgenerator/data.rb [hunk 1]
```diff
@@ -620,6 +620,15 @@ def osinfo
             'packaging_platform' => 'el-7-s390x'
           },
         },
+        'redhat7-AARCH64' => {
```
Comment (ScottGarman) [inline lib/beaker-hostgenerator/data.rb]: You'll also need to add an :abs entry for this platform. Please call it:

```
:abs -> {
  'template' => 'el-7-arm64'
}
```

File: lib/beaker-hostgenerator/data.rb [hunk 2]
```diff
@@ -620,6 +620,18 @@ def osinfo
...
+          :general => {
+            'platform'           => 'el-7-aarch64',
+            'packaging_platform' => 'el-7-aarch64'
+          },
+          :abs => {
+            'template' => 'el-7-arm64'
+          },
```
Comment (geoffnichols) [inline lib/beaker-hostgenerator/data.rb]: @ScottGarman Are the `:abs` entries currently required only for nspooler platforms?

Comment (ScottGarman) [reply lib/beaker-hostgenerator/data.rb]: @geoffnichols yes, that's my understanding.

**Comment by ScottGarman** [comment]
Did you re-generate the templates after the abs entry was added?

**Comment by caseywilliams** [comment]
Oops, no - completely forgot - I'll update.

**Pull request merged**